src/main/msc/emfat_file.c

```diff
@@ -45,10 +45,6 @@
 #define FILESYSTEM_SIZE_MB 256
 #define HDR_BUF_SIZE 32
 
-#define USE_EMFAT_AUTORUN
-#define USE_EMFAT_ICON
-//#define USE_EMFAT_README
```

Comment:
Maybe you can move this commented define in common_pre too

Reply:
I agree. We could also remove `USE_EMFAT_README` completely, looks like it's unused and doesn't really seem useful.